.travis.yml

```diff
@@ -1,22 +1,34 @@
 language: ruby
 script: "bundle exec rake"
 sudo: false
+gemfile:
+  - Gemfile
+  - gemfiles/faraday_0.13.gemfile
+  - gemfiles/faraday_0.12.gemfile
+  - gemfiles/faraday_0.11.gemfile
+  - gemfiles/faraday_0.10.gemfile
 rvm:
-  - 1.8.7
-  - 1.9.2
-  - 1.9.3
-  - 2.0.0
-  - 2.1.8
-  - 2.2.4
-  - 2.3.0
-  - 2.4.1
+  - 1.9
+  - 2.0
+  - 2.1
+  - 2.2
+  - 2.3
+  - 2.4
   - ruby-head
   - jruby-head
   - jruby-18mode
   - jruby-19mode
-matrix: 
+matrix:
   fast_finish: true
   allow_failures:
     - rvm: ruby-head
     - rvm: jruby-head
     - rvm: ree
+    - rvm: 1.9
```

Comment:
Appraisal is generating these gemfiles but they're using short hash syntax instead of arrows. I've got Travis running the usual `Gemfile` and marked these appraisal gemfiles as allowed failures to get around this. Its not nice, but it works.